src/Symfony/Component/Form/Form.php

```diff
@@ -727,11 +727,13 @@ public function isEmpty()
             }
         }
 
-        return FormUtil::isEmpty($this->modelData) ||
-            // arrays, countables
-            ((\is_array($this->modelData) || $this->modelData instanceof \Countable) && 0 === \count($this->modelData)) ||
-            // traversables that are not countable
-            ($this->modelData instanceof \Traversable && 0 === iterator_count($this->modelData));
+        if (!method_exists($this->config, 'getIsEmptyCallback')) {
+            @trigger_error(sprintf('Not implementing the getIsEmptyCallback() method in %s which implements %s is deprecated since Symfony 4.4.', \get_class($this->config), FormConfigInterface::class), E_USER_DEPRECATED);
+
+            return FormConfigBuilder::isEmpty($this->modelData);
```

Comment:
@xabbuh This is what I was talking about in https://github.com/symfony/symfony/pull/32747/files#r307712434

Reply:
We can remove `FormConfigBuilder::isEmpty()` in 5.0.

Reply:
👍 looks reasonable to me

Reply:
though we maybe need to trigger the deprecation in other places as well